src/GitHub.VisualStudio.UI/Views/InlineAnnotationView.xaml.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Windows.Controls;
+using GitHub.VisualStudio.UI.Helpers;
+
+namespace GitHub.VisualStudio.Views
+{
+    public partial class InlineAnnotationView : UserControl
+    {
+        public InlineAnnotationView()
+        {
+            InitializeComponent();
+            PreviewMouseWheel += ScrollViewerUtilities.FixMouseWheelScroll;
```

Comment:
Is this necessary here? I don't see a `ScrollViewer` in the XAML.

Reply:
I didn't understand what I was copying it seems.